guides/common/modules/snip_host-registration-steps.adoc

```diff
@@ -59,6 +59,11 @@ ifdef::katello,satellite,orcharhino[]
 -d '{ "registration_command": { "activation_keys": ["_My_Activation_Key_1_, _My_Activation_Key_2_"] }}'
 ----
 +
+[NOTE]
+====
+If your hosts do not trust the SSL certificate of {ProjectServer}, add the `--insecure` flag to the registration command.
```

Comment:
I think this only applied to the Hammer command. For the API request it needs to be inside the JSON data.

The whole idea is that you have:
1. Client -> Foreman: request command
2. Foreman -> Client: return command
3. Client -> Host: run command
4. Host -> Foreman: register itself

If you add `--insecure` to `curl`, that's step one, but you want it to be done in step 4, so you must request Foreman in step one to render the correct flag in step 2.

The original PR was the correct way to do that, so the Bugzilla report has the correct instructions.

Reply:
Means, we should go with this commit https://github.com/theforeman/foreman-documentation/commit/3a20d632ca9b1f18e44427a3958ee2bb46f20d0d ?

Reply:
That commit unconditionally specifies the parameter. I was suggesting to:
* For the CLI procedure: by default run without `--insecure 1`, but then document that it is an option. So move the note you have here to the CLI procedure.
* For the API procedure: provide examples where `insecure` is specified.

Katello (note compared to https://github.com/theforeman/foreman-documentation/commit/3a20d632ca9b1f18e44427a3958ee2bb46f20d0d it doesn't quote `true`):
```json
{ "registration_command": { "activation_keys": ["_My_Activation_Key_1_, _My_Activation_Key_2_"], "insecure": true }}
```

Non-Katello:
```json
{ "registration_command": { "insecure": true }}
```

Reply:
I can now see that `For the API procedure: provide examples` was ambiguous. What I meant was to first document the examples as they are today and then document you can make it insecure and then provide additional examples where it is insecure.

I know this makes it longer, but I think providing instructions on how to change the JSON is non-trivial and providing a secure example and an insecure example is better.